src/OpenProject.Revit/Entry/BcfierIpcHandler.cs

```diff
@@ -139,14 +141,20 @@ public void SendBringBrowserToForegroundRequestToDesktopApp()
     /// <summary>
     /// Same as in the windows app, but here we generate a VisInfo that is attached to the view
     /// </summary>
-    /// <param name="sender"></param>
-    /// <param name="e"></param>
+    /// <param name="trackingId">The local message tracking id.</param>
     private void AddView(string trackingId)
     {
       try
       {
-        var generatedViewpoint = RevitView.GenerateViewpoint(_uiapp.ActiveUIDocument);
-        var snapshot = GetRevitSnapshot(_uiapp.ActiveUIDocument.Document);
+        if (_uiApp.ActiveUIDocument.ActiveView.ViewType != ViewType.ThreeD)
```

Comment:
this check is the real thing. So, after receiving the event for viewpoint generation, we check for having a ui doc that has a 3D view. If not, we cannot generate a viewpoint and a dialog is displayed.